src/status_im/ui/components/connectivity/view.cljs

```diff
@@ -37,8 +37,7 @@
                                   (animation/timing anim-x (easing-out 0))])]))))}
     [react/view {:style {:width parent-width
                          :position :absolute
-                         :top -3
-                         :elevation 2
+                         :top -4
```

Comment:
I think it should be a -3 like before, if that's the progress bar then it should placed be on top of the separator and cover it.

Reply:
so the separator isn't visible when the progress bar is there?

Reply:
![Screenshot_1552299886](https://user-images.githubusercontent.com/1181225/54117057-504dd180-43f0-11e9-9e3e-9445f32b753a.png)

Reply:
![Screenshot_1552300350](https://user-images.githubusercontent.com/1181225/54117560-60b27c00-43f1-11e9-9530-10999a0728db.png)

Reply:
@errorists

Reply:
@yenda yeah but why is there a non transparent white background behind the progress bar? That's why I said to cover the separator, that it would still be visible.